pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -105,10 +118,88 @@ func (f *framework) RunReservePlugins(
 	return nil
 }
 
+// RunPermitPlugins runs the set of configured permit plugins. If any of these
+// plugins returns a status other than "Success" or "Wait", it does not continue
+// running the remaining plugins and returns an error. Otherwise, if any of the
+// plugins returns "Wait", then this function will block for the timeout period
```

Comment:
It would be good to clarify that if any plugin returns "wait", this function will continue running the remaining plugins.

Reply:
Done.